src/components/BShuttle/BShuttle.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-  <div class='shuttle'>
+  <div class="shuttle">
 
-    <div class='b-shuttle-available'>
+    <div class="b-shuttle-available">
       <label :id="`available-label-${id}`">{{ availableLabel }}</label>
       <ul
         :aria-labelledby="`available-label-${id}`"
         :aria-describedby="`placeholder-${id}`"
-        :tabindex="(placeholder && (!options || options.length === 0)) ? 0 : -1"
-        role='listbox'
-        class='input'
+        :tabindex="(placeholder && (!options || options.length === 0)) ? 0 : null"
+        role="listbox"
```

Comment:
We should also talk about this change. I thought we talked about writing strings with single quotes and then variables/bound options with double quotes? If not, we should make sure everyone is on the same page about what the pattern is.

Reply:
that was what we decided, yes.

Reply:
Why isn't everything one or the other? That's just confusing.

Reply:
@fobesjason per the poll in supreme-squadron, these strings should not be in double quotes.